apps/site/assets/css/_utilities.scss

```diff
@@ -57,6 +57,12 @@
 .u-bg--bus {
   @include bg-text($black);
   background-color: $brand-bus;
+  .c-link-block__inner-link {
+    border: 1px solid $black;
+    svg {
+      margin-top: 1px;
```

Comment:
Okay, figured out _why_ these cases would be different. 😵 Because this CSS in `_transit-near-me.scss` literally adds different spacing for the bus:

```scss
.m-tnm-sidebar__route-name {
  // ... omitted for brevity //

  // adjust icon colors and sizing based on background color
  @each $line in map-keys($mbta-line-colors) {
    &.u-bg--#{$line} {
      @include icon-mbta-colored($line, true);
      @if $line == 'bus' {
        @include icon-size-inline($base-spacing * 1.5, .275em);
      } @else {
        @include icon-size-inline($base-spacing * 1.5, .1em);
      }
    }
  }
  // ... //
}
```

it _could_ be worth looking into whether editing that would produce a desired result.

Reply:
Leaving these as is since tnm sidebar icons seem spaced appropriately. Made all other changes